ethpm_cli/commands/install.py

```diff
@@ -73,7 +76,9 @@ def format_for_display(self) -> str:
             alias = f" (alias: {self.path.name})"
         else:
             alias = ""
-        main_info = f"{prefix}{self.package_name}{alias}=={self.package_version}"
+        blue_package_name = f"{BLUE_STRING}{self.package_name}{COLOR_RESET}"
```

Comment:
I'd recommend utility functions for this.

```python
def blue(text):
    return f"{BLUE_STRING}{text}{COLOR_RESET}"
```
blue_package_name = blue(self.package_name)

Or you can use the `termcolor` package like we do in trinity: https://github.com/ethereum/trinity/blob/master/trinity/_utils/shellart.py